packages/storybook/stories/components/text/text.stories.ts

```diff
@@ -105,3 +106,52 @@ export const CustomCSS: StoryObj = {
 </style>
   `,
 };
+
+export const CaptionTable: StoryObj = {
+  tags: ['isHidden'],
+  render: () => html`
+<table>
+  <caption>
+    <ods-text preset="caption">My table title</ods-text>
+  </caption>
+  <thead>
+  <tr>
+    <th scope="col">Person</th>
+    <th scope="col">Age</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <th scope="row">Chris</th>
+    <td>22</td>
+  </tr>
+  </tbody>
+</table>
+<style>
+  caption {
+    caption-side: bottom;
+  }
+  table {
```

Comment:
use a specific uniq id as this is apply to the whole page => this specifically apply on the `Properties` table on top of the page.